content/en/news/releases/1.7.x/announcing-1.7/change-notes/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Change Notes
+description: Istio 1.7 release notes.
+weight: 10
+---
+
+## Traffic Management
+
+- **Added** config option `values.global.proxy.holdApplicationUntilProxyStarts`,
+which causes the sidecar injector to inject the sidecar at the start of the
+pod's container list and configures it to block the start of all other
+containers until the proxy is ready.  This option is disabled by default.
+ ([Issue #11130](https://github.com/istio/istio/issues/11130))
```

Comment:
Is our list or tool correct?  I'm fairly sure we did more than this 1 item in 1.7 for traffic mgmt.  cc @brian-avery

Reply:
Hey @linsun [this](https://github.com/istio/istio/blob/release-1.7/releasenotes/notes/24737.yaml) was the only one created for 1.7 with an area of traffic management.

Reply:
Master has: 
```
24737.yaml
25746.yaml
25818.yaml
25678.yaml
```

Reply:
Looks like [26001.yaml](https://github.com/williamaronli/istio/blob/release-1.7/releasenotes/notes/26001.yaml) added an "area: pilot". This isn't listed anywhere in the template. Everything else uses one of the specified areas. 

```
grep -h 'area:' * | sort | uniq
area: documentation
area: installation
area: istioctl
area: networking
area: pilot
area: security
area: telemetry
area: traffic-management
```

Reply:
hmm, Also, there are ones under networking, e.g. https://github.com/istio/istio/blob/master/releasenotes/notes/add-sni-host.yaml.  Why do we have these many categories (networking, pilot, traffic management)?  Shouldn't be all under traffic management or Networking?

Reply:
You're right. Looking at the 1.6 release notes, we had: 
    Traffic Management
    Security
    Telemetry
    Installation
    istioctl
    Documentation changes

So the networking and pilot ones are both excluded right now.

Reply:
So 
```
dd-sni-host.yaml:area: networking
auto-mtls-headless.yaml:area: networking
delay-app-start.yaml:area: networking
dns-capture.yaml:area: networking
dr-analyzer.yaml:area: networking
dr-sds.yaml:area: networking
endpoint-before-pod.yaml:area: networking
ingress-named-port.yaml:area: networking
service-select-workload-entry.yaml:area: networking
xds-v3.yaml:area: networking
```

Are all missing.

Reply:
DNS capture is something @rshriram removed recently.  let's remove it off the release note to avoid confusion.